### DISCUSSION TITLE
without pre format we need to write ISO file into CD/DVD

### DISCUSSION THREAD

**Discussion by Avinash-c-gowda** [nroduit/Weasis#476] [Ideas]
Hi @nroduit,

1) If it is an empty CD/DVD, we still need to format it before we can write an ISO file onto it. Is it possible to write to a CD/DVD without pre-formatting?

2) If the file system is UDF, we need to use external software for mounting. Otherwise, we may encounter errors. 
  My idea is to write the .ISO as a CDFS file system onto the CD/DVD. Whether the file system is UDF, NTFS, or any other, we need to convert it to CDFS to view the content directly on the CD. Is that possible?

Thanks And Regards
Avinash

**Comment by nroduit** [comment]
The Weasis implementation is aligned with the [DICOM standard ISO 9660 File System](https://dicom.nema.org/MEDICAL/Dicom/current/output/chtml/part12/sect_P.2.2.html) with Rockridge or Joliet Extensions.

This functionality is no longer really relevant to the way data is shared, and will certainly not evolve.

**Comment by Avinash-c-gowda** [comment]
@nroduit, but every time we need to format the empty CD also, then only it is exporting. how to stop that

**Comment by nroduit** [comment]
As mentioned above, the iso build is based on the DICOM standard. If there is a problem in the implementation of this standard, please describe the problem precisely. Otherwise, the implementation of other standards or file systems is not in my scope.

**Reply by Avinash-c-gowda** [reply]
ok, Thanks
